### PR TITLE
[6494] Changed user droplist (now takes data from xml). Cron user update

### PR DESCRIPTION
**Pull request by ADubielSTX**
**Story / Bug id**: https://bugzilla.stxnext.pl/show_bug.cgi?id=6494
**Project / App**: Onboarding - Presence Analyzer
**Reference person**: @ADubielSTX
**Description**: Changing users droplist (now displays user name and last name instead of User <id>). Taking user info from xml file and auto updating that file (cron).

**Migrations**:

**Rollback**:

**New imports / dependencies**: 

**Unit Tests**:

**What tests do I need to run to validate this change**:

## Scripts to be run:
./bin/buildout -vN
(to update xml file cofing and update crontab)

## New or Changed JS and / or HTML Files:
changed:
src/presence_analyzer/layout.html
src/presence_analyzer/static/mean_time_weekday.js
src/presence_analyzer/static/presence_start_end.js
src/presence_analyzer/static/presence_weekday.js
new:
update_avatar.js

### CODE REVIEW

**Comment by ADubielSTX** [comment]
Gist with pep8, pylint, unittests etc.:
https://gist.github.com/ADubielSTX/a2f246a7ca6da3f04651

File: src/presence_analyzer/tests.py [hunk 1]
```diff
@@ -64,7 +70,10 @@ def test_api_users(self):
         self.assertEqual(resp.content_type, 'application/json')
         data = json.loads(resp.data)
         self.assertEqual(len(data), 2)
-        self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
+        self.assertDictEqual(data[0], {
+            u'user_id': '11',
```
Comment (piasekps) [inline src/presence_analyzer/tests.py]: Once you import this from 
__future__ import unicode_literals 
all string will be by default unicode and you will not need u before string.

File: src/presence_analyzer/utils.py [hunk 2]
```diff
@@ -71,6 +73,40 @@ def get_data():
...
+            'name': 'Adrian K.',
+        },
+    }
+    """
+    data = {}
+    with open(app.config['DATA_XML'], 'r') as xmlfile:
+        xmldata_tree_object = etree.parse(xmlfile)
+        data_xml = xmldata_tree_object.getroot()
+
+        server_data = {server.tag: server.text for server in data_xml[0]}
```
Comment (piasekps) [inline src/presence_analyzer/utils.py]: :+1:

File: src/presence_analyzer/utils.py [hunk 3]
```diff
@@ -71,6 +73,40 @@ def get_data():
...
+        base_path = server_data['protocol'] + '://' + server_data['host']
```
Comment (piasekps) [inline src/presence_analyzer/utils.py]: can you use format to manipulate with string?

File: src/presence_analyzer/utils.py (hunk 2 above)
Comment (piasekps) [inline src/presence_analyzer/utils.py]: It's nice that you use dict comprehension, but etree allow you much easier find elements than parsing it by yourself. Please use method build in lxml etree.

**Comment by piasekps** [comment]
Review finished

**Pull request merged**